coilsnake/model/common/ips.py

```diff
@@ -128,7 +128,8 @@ def create(self, clean_rom, hacked_rom, patch_path):
                             records[i] = t
                         else:
                             i -= 1
-                            records[i] = hr.to_list()[i]
+                            index -= 1
+                            records[i] = hr[i:i+1].to_array().tobytes()
```

Comment:
IMO we should use the same method as we normally do to get the byte:
```suggestion
                            records[i] = hr.__getitem__(i).to_bytes(1, byteorder='big')
```

(Not that I like the use of `__getitem__` here, but I think we should keep it consistent w. the other lines which get the value of `t`)